.travis.yml

```diff
@@ -21,8 +21,10 @@ script:
 cache: npm
 
 # Thinking about adding install steps? Why not modify the tools/setup script instead?
+# The list command dumps the top-level packages with their versions, in case we need that for debugging.
 install:
   - ./tools/setup
```

Comment:
Can you also update the setup script, see:

https://github.com/PolymerLabs/arcs/blob/master/tools/setup#L85

and L88

Reply:
Never mind... it's in here.